plugins/inventory/now.py

```diff
@@ -442,8 +454,23 @@ def set_hostvars(self, host, record, columns):
                 "Invalid column names: {0}.".format(", ".join(missing))
             )
 
+        show_values = self.get_option("show_values")
+
         for k in columns:
-            self.inventory.set_variable(host, k.replace(".", "_"), record[k])
+            value = record.get(k)
+            if isinstance(value, dict):
```

Comment:
what if the value is not a dict? Where do you set the value to host_var if value is str?